docs/better.html

```diff
@@ -9,10 +9,13 @@
   <link rel="stylesheet" href="main.css"></link>
 </head>
 <body>
+  <!-- htmllint attr-bans="false" attr-name-style="false" -->
   <svg id="logo" xmlns="http://www.w3.org/2000/svg" version="1.1" viewBox="0 0 100 100">
     <circle cx="50%" cy="50%" r="50%" fill="rgba(82, 14, 33, 0.2)" />
     <use href="firefox-logo.svg#Firefox_-_Large" x="5%" y="5%" width="90%" height="90%" />
   </svg>
+  <!-- htmllint attr-bans="$previous" attr-name-style="$previous" -->
```

Comment:
Not a huge fan of inline disabling/enabling of rules (especially since we have it duplicated in 2 files), but not sure if there's an easier `<img src="firefox-logo.svg" />` type solution.